raspberry/bzzz/sensors/TeraBee_Evo_time_of_flight_sensor/Evo_single_point_display_range_py3.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+###### TeraRanger Evo Example Code STD #######
+#                                            #
+# All rights reserved Terabee France (c) 2018#
+#                                            #
+############ www.terabee.com #################
+
+import serial
+import serial.tools.list_ports
+import sys
+import crcmod.predefined  # To install: pip install crcmod
+
+
+def findEvo():
+    # Find Live Ports, return port name if found, NULL if not
+    print('Scanning all live ports on this PC')
+    ports = list(serial.tools.list_ports.comports())
+    for p in ports:
+        # print p # This causes each port's information to be printed out.
+        if "5740" in p[2]:
+            print('Evo found on port ' + p[0])
+            return p[0]
+    return 'NULL'
+
+
+def openEvo(portname):
```

Comment:
Function names should be snake-case (see [here](https://github.com/QUB-ASL/bzzz/blob/main/CONTRIBUTING.md#32-python))